src/Symfony/Component/HttpKernel/DataCollector/RequestDataCollector.php

```diff
@@ -263,11 +263,21 @@ public function getContent()
         return $this->data['content'];
     }
 
+    public function getPrettyJsonContent()
```

Comment:
i think we need to use json_last_error() indeed, as "null" can be a valid JSON value too: https://3v4l.org/I7DM7

i propose the return a structure like `return array('content' => json_encode(...), 'valid' => true/false);` and rename the method to `getPrettyJson()`